### PR TITLE
Gardener proposal

### PR DESCRIPTION
**Pull request by kss-espeo**
As per https://wiki.hyperledger.org/display/HYP/Gardener

### CODE REVIEW

**Comment by lehors** [comment]
DCO needs fixing but otherwise, it looks good to me.

**Comment by kss-espeo** [comment]
> DCO needs fixing but otherwise, it looks good to me.

Done.

File: proposal-template.md [hunk 1]
```diff
@@ -1,22 +0,0 @@
-# Lab Name
```
Comment (trbs) [inline proposal-template.md]: Please do not remove the `proposal-template.md` file :-)

**Comment by kss-espeo** [comment]
> Please do not remove the `proposal-template.md` file :-)

Well, this is awkward... done.

**Comment by tkuhrt** [comment]
Looks like commit this morning broke DCO again. Can you update the commit to include the sign off?

**Comment by tkuhrt** [comment]
@dcmiddle  : Can you confirm sponsorship of this lab?

**Comment by kss-espeo** [comment]
Closing this one, will create new fork and PR because I can't seem to fix DCO with current commit history (multiple merges from master)

**Pull request closed without merging**